### PR TITLE
DIV-8098 | Retrieve ICD11 mapping from etcd configuration layer

### PR DESCRIPTION
**Pull request by varadeth**

### CODE REVIEW

File: backend/certificate_signer/config/constants.js [hunk 1]
```diff
@@ -1,134 +1,9 @@
...
-    "vaccineType": "Protein subunit",
-    "icd11Term": "COVID-19 vaccine, virus protein subunit",
-    "disease": "COVID-19"
-  },
-  "XM1J92": {
-    "vaccineType": "Virus like particle",
-    "icd11Term": "COVID-19 vaccine, virus like particle (VLP)",
-    "disease": "COVID-19"
-  }
+const MAPPINGS_KEYS = {
```
Comment (egov-joy) [inline backend/certificate_signer/config/constants.js:1]: Rename this so it is clear what mappings these are.

File: docker-compose.yml [hunk 2]
```diff
@@ -207,6 +207,8 @@ services:
       CERTIFICATE_PRIVATE_KEY: "$CERTIFICATE_SIGNER_PRIVATE_KEY"
       COUNTRY_CODE: "IND"
       FACILITY_COUNTRY_CODE: "IND"
+      ETCD_URL: etcd:2379
```
Comment (egov-joy) [inline docker-compose.yml:210]: Can we refer to the env variable for port here?

File: docker-compose.e2e.yml [hunk 3]
```diff
@@ -10,6 +10,10 @@ services:
   portal_api:
     environment:
         ADMIN_API_CLIENT_SECRET: "$ADMIN_API_CLIENT_SECRET"
+  etcd:
+    entrypoint: /usr/local/bin/etcd
+    environment:
+      ETCD_ENABLE_V2: "true"
```
Comment (egov-joy) [inline docker-compose.e2e.yml:16]: Is this required for backward compatibility to run the tests?

**Review by egov-joy — COMMENTED** [review]
Added comments.

File: docker-compose.yml (hunk 2 above)
Comment (varadeth) [reply docker-compose.yml:210]: I am thinking to move whole url part to env. it is a good way?

File: docker-compose.e2e.yml (hunk 3 above)
Comment (varadeth) [reply docker-compose.e2e.yml:16]: I tried v2 approach for tests but that is not required. I will remove this

**Pull request merged**